anchors/service_test.go

```diff
@@ -21,10 +21,20 @@ type mockAnchorRepo struct {
 	anchorRepositoryContract
 }
 
-func (m *mockAnchorRepo) Commits(opts *bind.CallOpts, anchorID *big.Int) (docRoot [32]byte, err error) {
+func (m *mockAnchorRepo) GetAnchorById(opts *bind.CallOpts, anchorID *big.Int) (struct {
+	AnchorID     *big.Int
+	DocumentRoot [32]byte
+}, error) {
 	args := m.Called(opts, anchorID)
-	docRoot, _ = args.Get(0).([32]byte)
-	return docRoot, args.Error(1)
+	type Response struct {
```

Comment:
once the struct is created, you can pass the struct as is instead of creating one here